src/hotspot/share/opto/compile.hpp

```diff
@@ -1173,11 +1173,7 @@ class Compile : public Phase {
   bool           in_scratch_emit_size() const   { return _in_scratch_emit_size;     }
 
   enum ScratchBufferBlob {
-#if defined(PPC64)
```

Comment:
why delete?

Reply:
@kuaiwei is it necessary to port new `MAX_inst_size` value?

Reply:
I think it's ok to use the new MAX_inst_size value